### ISSUE TITLE
When clicking the location button on chat fragments the app crashes

### ISSUE THREAD

**Issue by karimkod** [dimelo/Dimelo-Android#43]
So I'm using the chat fragment and when I click the location button it crashes the app, knowing that I'm using my key for mapsApi and I have another use of maps (outside dimelo) and it works perfectly 

Thanks.

**Comment by waelba** [comment]
Hi,

Could you please share your gradle file and your application logs when you try to open Maps via the Dimelo Mobile SDK ?

Best regards,

**Comment by karimkod** [comment]
Gradle : 

`apply plugin: 'com.android.application'
apply plugin: 'io.fabric'

apply plugin: 'kotlin-android'

apply plugin: 'kotlin-android-extensions'

apply plugin: 'org.jetbrains.dokka-android'



android {
    compileSdkVersion 27
    defaultConfig {
        applicationId "com.brenco.citylockerapp"
        minSdkVersion 19
        targetSdkVersion 27
        versionCode 1
        versionName "1.0.7"
        testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
    }
    signingConfigs {

        release {
            storeFile file(RELEASE_STORE_FILE)
            storePassword RELEASE_STORE_PASSWORD
            keyAlias RELEASE_KEY_ALIAS
            keyPassword RELEASE_KEY_PASSWORD
        }
    }

    buildTypes {
        release {
            minifyEnabled false
            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
            signingConfig signingConfigs.release
            ext.betaDistributionReleaseNotes="Tester l'inscription S.V.P"
            ext.betaDistributionEmails="kbournane@brenco-algerie.com, kbrouri@brenco-algerie.com, babtroun@brenco-algerie.com, sboudjema@brenco-algerie.com, ybouhadjer@brenco-Algerie.com, sdjouadi@brenco-algerie.com"
        }


    }

}


android.buildTypes.each { type ->
    type.buildConfigField 'String', 'GCM_API_KEY', GCM_API_KEY
    type.buildConfigField 'String', 'DIMELO_SDK_SECRET', DIMELO_SDK_SECRET
}

repositories {

    maven {
        url "https://raw.github.com/dimelo/Dimelo-Android/master"
    }
    maven { url 'https://maven.fabric.io/public' }


}


dokka {
    outputFormat = 'markdown'
    outputDirectory = "$buildDir/documentation"

    includes = ['Module.md']
}

dependencies {
    implementation fileTree(dir: 'libs', include: ['*.jar'])
    implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
    implementation 'com.android.support:appcompat-v7:27.1.1'
    implementation 'com.android.support.constraint:constraint-layout:1.1.3'
    implementation 'com.google.firebase:firebase-core:16.0.5'
    implementation 'com.google.firebase:firebase-auth:16.0.5'
    implementation 'com.dimelo.dimelosdk:dimelosdk:1.7.0'
    //implementation 'com.google.android.gms:play-services-gcm:16.0.0'
    implementation "com.google.firebase:firebase-messaging:17.3.4"

    implementation 'com.google.gms:google-services:3.2.0'
    implementation 'com.squareup.retrofit2:retrofit:2.4.0'
    implementation 'com.google.firebase:firebase-database:16.0.4'
    implementation 'com.google.firebase:firebase-storage:16.0.4'
    implementation 'com.github.davidmigloz:youtube-android-player-api-gradle:1.2.2.1'
    implementation('com.crashlytics.sdk.android:crashlytics:2.9.4@aar') {
        transitive = true;
    }

    //design
    implementation 'com.android.support:cardview-v7:27.1.1'
    implementation 'com.android.support:design:27.1.1'
    implementation 'com.android.support:support-v4:27.1.1'
    implementation 'com.google.android.gms:play-services-maps:16.0.0'
    testImplementation 'junit:junit:4.12'
    androidTestImplementation 'com.android.support.test:runner:1.0.2'
    androidTestImplementation 'com.android.support.test.espresso:espresso-core:3.0.2'


}



apply plugin: 'com.google.gms.google-services'
`

Log  : 

`11-26 14:32:43.416 2919-3010/? I/PowerManagerService: canDreamLocked:mCharingBase=false
    canDreamLocked:mChargingMode=0
11-26 14:32:43.456 29482-29482/com.brenco.citylockerapp I/dalvikvm: Could not find method android.app.Activity.finishAfterTransition, referenced from method android.support.v4.app.ActivityCompat.finishAfterTransition
11-26 14:32:43.456 29482-29482/com.brenco.citylockerapp W/dalvikvm: VFY: unable to resolve virtual method 137: Landroid/app/Activity;.finishAfterTransition ()V
11-26 14:32:43.456 29482-29482/com.brenco.citylockerapp I/dalvikvm: Could not find method android.app.Activity.getReferrer, referenced from method android.support.v4.app.ActivityCompat.getReferrer
11-26 14:32:43.456 29482-29482/com.brenco.citylockerapp W/dalvikvm: VFY: unable to resolve virtual method 151: Landroid/app/Activity;.getReferrer ()Landroid/net/Uri;
11-26 14:32:43.456 29482-29482/com.brenco.citylockerapp I/dalvikvm: Could not find method android.app.Activity.postponeEnterTransition, referenced from method android.support.v4.app.ActivityCompat.postponeEnterTransition
11-26 14:32:43.456 29482-29482/com.brenco.citylockerapp W/dalvikvm: VFY: unable to resolve virtual method 171: Landroid/app/Activity;.postponeEnterTransition ()V
11-26 14:32:43.456 29482-29482/com.brenco.citylockerapp I/dalvikvm: Could not find method android.app.Activity.requestPermissions, referenced from method android.support.v4.app.ActivityCompat.requestPermissions
11-26 14:32:43.456 29482-29482/com.brenco.citylockerapp W/dalvikvm: VFY: unable to resolve virtual method 174: Landroid/app/Activity;.requestPermissions ([Ljava/lang/String;I)V
    Unable to resolve superclass of Landroid/support/v4/app/ActivityCompat$SharedElementCallback21Impl; (76)
    Link of class 'Landroid/support/v4/app/ActivityCompat$SharedElementCallback21Impl;' failed
    Unable to resolve superclass of Landroid/support/v4/app/ActivityCompat$SharedElementCallback23Impl; (1043)
    Link of class 'Landroid/support/v4/app/ActivityCompat$SharedElementCallback23Impl;' failed
11-26 14:32:43.456 29482-29482/com.brenco.citylockerapp E/dalvikvm: Could not find class 'android.support.v4.app.ActivityCompat$SharedElementCallback23Impl', referenced from method android.support.v4.app.ActivityCompat.setEnterSharedElementCallback
11-26 14:32:43.456 29482-29482/com.brenco.citylockerapp W/dalvikvm: VFY: unable to resolve new-instance 1045 (Landroid/support/v4/app/ActivityCompat$SharedElementCallback23Impl;) in Landroid/support/v4/app/ActivityCompat;
11-26 14:32:43.456 29482-29482/com.brenco.citylockerapp I/dalvikvm: Could not find method android.app.Activity.setEnterSharedElementCallback, referenced from method android.support.v4.app.ActivityCompat.setEnterSharedElementCallback
11-26 14:32:43.456 29482-29482/com.brenco.citylockerapp W/dalvikvm: VFY: unable to resolve virtual method 176: Landroid/app/Activity;.setEnterSharedElementCallback (Landroid/app/SharedElementCallback;)V
    Unable to resolve superclass of Landroid/support/v4/app/ActivityCompat$SharedElementCallback21Impl; (76)
    Link of class 'Landroid/support/v4/app/ActivityCompat$SharedElementCallback21Impl;' failed
11-26 14:32:43.456 29482-29482/com.brenco.citylockerapp E/dalvikvm: Could not find class 'android.support.v4.app.ActivityCompat$SharedElementCallback21Impl', referenced from method android.support.v4.app.ActivityCompat.setEnterSharedElementCallback
11-26 14:32:43.456 29482-29482/com.brenco.citylockerapp W/dalvikvm: VFY: unable to resolve new-instance 1043 (Landroid/support/v4/app/ActivityCompat$SharedElementCallback21Impl;) in Landroid/support/v4/app/ActivityCompat;
11-26 14:32:43.461 29482-29482/com.brenco.citylockerapp I/dalvikvm: Could not find method android.app.Activity.setEnterSharedElementCallback, referenced from method android.support.v4.app.ActivityCompat.setEnterSharedElementCallback
11-26 14:32:43.461 29482-29482/com.brenco.citylockerapp W/dalvikvm: VFY: unable to resolve virtual method 176: Landroid/app/Activity;.setEnterSharedElementCallback (Landroid/app/SharedElementCallback;)V
    Unable to resolve superclass of Landroid/support/v4/app/ActivityCompat$SharedElementCallback21Impl; (76)
    Link of class 'Landroid/support/v4/app/ActivityCompat$SharedElementCallback21Impl;' failed
    Unable to resolve superclass of Landroid/support/v4/app/ActivityCompat$SharedElementCallback23Impl; (1043)
    Link of class 'Landroid/support/v4/app/ActivityCompat$SharedElementCallback23Impl;' failed
11-26 14:32:43.461 29482-29482/com.brenco.citylockerapp E/dalvikvm: Could not find class 'android.support.v4.app.ActivityCompat$SharedElementCallback23Impl', referenced from method android.support.v4.app.ActivityCompat.setExitSharedElementCallback
11-26 14:32:43.461 29482-29482/com.brenco.citylockerapp W/dalvikvm: VFY: unable to resolve new-instance 1045 (Landroid/support/v4/app/ActivityCompat$SharedElementCallback23Impl;) in Landroid/support/v4/app/ActivityCompat;
11-26 14:32:43.461 29482-29482/com.brenco.citylockerapp I/dalvikvm: Could not find method android.app.Activity.setExitSharedElementCallback, referenced from method android.support.v4.app.ActivityCompat.setExitSharedElementCallback
11-26 14:32:43.461 29482-29482/com.brenco.citylockerapp W/dalvikvm: VFY: unable to resolve virtual method 177: Landroid/app/Activity;.setExitSharedElementCallback (Landroid/app/SharedElementCallback;)V
    Unable to resolve superclass of Landroid/support/v4/app/ActivityCompat$SharedElementCallback21Impl; (76)
    Link of class 'Landroid/support/v4/app/ActivityCompat$SharedElementCallback21Impl;' failed
11-26 14:32:43.461 29482-29482/com.brenco.citylockerapp E/dalvikvm: Could not find class 'android.support.v4.app.ActivityCompat$SharedElementCallback21Impl', referenced from method android.support.v4.app.ActivityCompat.setExitSharedElementCallback
11-26 14:32:43.461 29482-29482/com.brenco.citylockerapp W/dalvikvm: VFY: unable to resolve new-instance 1043 (Landroid/support/v4/app/ActivityCompat$SharedElementCallback21Impl;) in Landroid/support/v4/app/ActivityCompat;
11-26 14:32:43.461 29482-29482/com.brenco.citylockerapp I/dalvikvm: Could not find method android.app.Activity.setExitSharedElementCallback, referenced from method android.support.v4.app.ActivityCompat.setExitSharedElementCallback
11-26 14:32:43.461 29482-29482/com.brenco.citylockerapp W/dalvikvm: VFY: unable to resolve virtual method 177: Landroid/app/Activity;.setExitSharedElementCallback (Landroid/app/SharedElementCallback;)V
11-26 14:32:43.461 29482-29482/com.brenco.citylockerapp I/dalvikvm: Could not find method android.app.Activity.shouldShowRequestPermissionRationale, referenced from method android.support.v4.app.ActivityCompat.shouldShowRequestPermissionRationale
11-26 14:32:43.461 29482-29482/com.brenco.citylockerapp W/dalvikvm: VFY: unable to resolve virtual method 180: Landroid/app/Activity;.shouldShowRequestPermissionRationale (Ljava/lang/String;)Z
11-26 14:32:43.461 29482-29482/com.brenco.citylockerapp I/dalvikvm: Could not find method android.app.Activity.startPostponedEnterTransition, referenced from method android.support.v4.app.ActivityCompat.startPostponedEnterTransition
11-26 14:32:43.461 29482-29482/com.brenco.citylockerapp W/dalvikvm: VFY: unable to resolve virtual method 187: Landroid/app/Activity;.startPostponedEnterTransition ()V
11-26 14:32:43.466 29482-29482/com.brenco.citylockerapp W/dalvikvm: Unable to resolve superclass of Landroid/support/v4/app/ActivityCompat$SharedElementCallback21Impl; (76)
    Link of class 'Landroid/support/v4/app/ActivityCompat$SharedElementCallback21Impl;' failed
    Unable to resolve superclass of Landroid/support/v4/app/ActivityCompat$SharedElementCallback23Impl; (1043)
    Link of class 'Landroid/support/v4/app/ActivityCompat$SharedElementCallback23Impl;' failed
    Unable to resolve superclass of Landroid/support/v4/app/ActivityCompat$SharedElementCallback21Impl; (76)
    Link of class 'Landroid/support/v4/app/ActivityCompat$SharedElementCallback21Impl;' failed
    Unable to resolve superclass of Landroid/support/v4/app/ActivityCompat$SharedElementCallback21Impl; (76)
    Link of class 'Landroid/support/v4/app/ActivityCompat$SharedElementCallback21Impl;' failed
    Unable to resolve superclass of Landroid/support/v4/app/ActivityCompat$SharedElementCallback23Impl; (1043)
    Link of class 'Landroid/support/v4/app/ActivityCompat$SharedElementCallback23Impl;' failed
11-26 14:32:43.471 29482-29482/com.brenco.citylockerapp W/dalvikvm: Unable to resolve superclass of Landroid/support/v4/app/ActivityCompat$SharedElementCallback21Impl; (76)
    Link of class 'Landroid/support/v4/app/ActivityCompat$SharedElementCallback21Impl;' failed
11-26 14:32:43.521 2919-3010/? I/PowerManagerService: canDreamLocked:mCharingBase=false
    canDreamLocked:mChargingMode=0
11-26 14:32:43.556 29482-29482/com.brenco.citylockerapp I/Google Maps Android API: Google Play services package version: 14574004
11-26 14:32:43.606 29482-29482/com.brenco.citylockerapp I/dalvikvm: Could not find method android.content.pm.PackageManager.isInstantApp, referenced from method com.google.maps.api.android.lib6.drd.aj.a
11-26 14:32:43.606 29482-29482/com.brenco.citylockerapp W/dalvikvm: VFY: unable to resolve virtual method 582: Landroid/content/pm/PackageManager;.isInstantApp (Ljava/lang/String;)Z
11-26 14:32:43.696 29482-29482/com.brenco.citylockerapp I/dalvikvm: Could not find method android.content.pm.PackageManager.isInstantApp, referenced from method nu.a
11-26 14:32:43.696 29482-29482/com.brenco.citylockerapp W/dalvikvm: VFY: unable to resolve virtual method 581: Landroid/content/pm/PackageManager;.isInstantApp ()Z
11-26 14:32:43.746 29482-29482/com.brenco.citylockerapp I/dalvikvm: Could not find method android.app.NotificationManager.getNotificationChannel, referenced from method hq.a
11-26 14:32:43.746 29482-29482/com.brenco.citylockerapp W/dalvikvm: VFY: unable to resolve virtual method 494: Landroid/app/NotificationManager;.getNotificationChannel (Ljava/lang/String;)Landroid/app/NotificationChannel;
11-26 14:32:43.756 29482-29482/com.brenco.citylockerapp I/dalvikvm: Could not find method android.content.Context.createDeviceProtectedStorageContext, referenced from method sy.a
11-26 14:32:43.756 29482-29482/com.brenco.citylockerapp W/dalvikvm: VFY: unable to resolve virtual method 507: Landroid/content/Context;.createDeviceProtectedStorageContext ()Landroid/content/Context;
11-26 14:32:43.756 29482-29482/com.brenco.citylockerapp I/dalvikvm: Could not find method android.app.AppOpsManager.permissionToOp, referenced from method sr.a
11-26 14:32:43.756 29482-29482/com.brenco.citylockerapp W/dalvikvm: VFY: unable to resolve static method 404: Landroid/app/AppOpsManager;.permissionToOp (Ljava/lang/String;)Ljava/lang/String;
11-26 14:32:43.756 29482-29482/com.brenco.citylockerapp I/dalvikvm: Could not find method android.content.Context.getSystemService, referenced from method sr.a
11-26 14:32:43.756 29482-29482/com.brenco.citylockerapp W/dalvikvm: VFY: unable to resolve virtual method 524: Landroid/content/Context;.getSystemService (Ljava/lang/Class;)Ljava/lang/Object;
11-26 14:32:43.901 29482-29482/com.brenco.citylockerapp I/dalvikvm: Could not find method android.view.accessibility.AccessibilityNodeInfo.setScreenReaderFocusable, referenced from method dd.onInitializeAccessibilityNodeInfo
11-26 14:32:43.901 29482-29482/com.brenco.citylockerapp W/dalvikvm: VFY: unable to resolve virtual method 1236: Landroid/view/accessibility/AccessibilityNodeInfo;.setScreenReaderFocusable (Z)V
11-26 14:32:43.901 29482-29482/com.brenco.citylockerapp I/dalvikvm: Could not find method android.view.accessibility.AccessibilityNodeInfo.setHeading, referenced from method dd.onInitializeAccessibilityNodeInfo
11-26 14:32:43.901 29482-29482/com.brenco.citylockerapp W/dalvikvm: VFY: unable to resolve virtual method 1231: Landroid/view/accessibility/AccessibilityNodeInfo;.setHeading (Z)V
11-26 14:32:43.901 29482-29482/com.brenco.citylockerapp I/dalvikvm: Could not find method android.view.accessibility.AccessibilityNodeInfo.setPaneTitle, referenced from method dd.onInitializeAccessibilityNodeInfo
11-26 14:32:43.901 29482-29482/com.brenco.citylockerapp W/dalvikvm: VFY: unable to resolve virtual method 1233: Landroid/view/accessibility/AccessibilityNodeInfo;.setPaneTitle (Ljava/lang/CharSequence;)V
11-26 14:32:43.901 29482-29482/com.brenco.citylockerapp I/dalvikvm: Could not find method android.view.View.setTransitionName, referenced from method dh.a
11-26 14:32:43.901 29482-29482/com.brenco.citylockerapp W/dalvikvm: VFY: unable to resolve virtual method 1143: Landroid/view/View;.setTransitionName (Ljava/lang/String;)V
11-26 14:32:43.901 29482-29482/com.brenco.citylockerapp I/dalvikvm: Could not find method android.view.View.getTransitionName, referenced from method dh.c
11-26 14:32:43.901 29482-29482/com.brenco.citylockerapp W/dalvikvm: VFY: unable to resolve virtual method 1109: Landroid/view/View;.getTransitionName ()Ljava/lang/String;
11-26 14:32:43.916 29482-29482/com.brenco.citylockerapp I/dalvikvm: Could not find method com.google.android.gms.common.internal.zzbo.zzu, referenced from method com.google.android.gms.location.places.ui.PlacePicker$IntentBuilder.setLatLngBounds
11-26 14:32:43.916 29482-29482/com.brenco.citylockerapp W/dalvikvm: VFY: unable to resolve static method 31946: Lcom/google/android/gms/common/internal/zzbo;.zzu (Ljava/lang/Object;)Ljava/lang/Object;
11-26 14:32:43.916 29482-29482/com.brenco.citylockerapp I/dalvikvm: Could not find method com.google.android.gms.common.GoogleApiAvailability.zzas, referenced from method com.google.android.gms.location.places.ui.zzb.build
11-26 14:32:43.916 29482-29482/com.brenco.citylockerapp W/dalvikvm: VFY: unable to resolve static method 29659: Lcom/google/android/gms/common/GoogleApiAvailability;.zzas (Landroid/content/Context;)V
    threadid=1: thread exiting with uncaught exception (group=0x41b6ec50)
11-26 14:32:44.101 2429-2504/? E/logserver: Object Path:/data/system/dropbox/, mask=0x00000080
    event->len=48, name=data_app_crash@1543239164106.txt
    find_first_match: find match, plogs(type=1, pfile=/data/system/dropbox/, match=app_crash)
11-26 14:32:44.106 2429-2504/? E/logserver: move_dir:src_path=/data/log/dbox/appdbg/logcache
    move_dir:dst_path=/data/log/logcache/33592936/
    is_all_ready, any file is ready in non-keyfile exception.
    extract_appname, forward search, appname=com.brenco.citylockerapp
    archive_and_send, pos=0, type=crash, output=20181126143244_crash
    ---copy_match_files enter!!--
    [copy_match_files,829]: copy [/data/system/dropbox/data_app_crash@1543239164106.txt] to [/data/log/logcache/33592936/data_app_crash@1543239164106.txt]
    internal_archive, argc=7
    get_fault_appname, appname=com.brenco.citylockerapp
    handle_archive_exception, argc=7
    get alarm time = 818459 (sec.)!
    log_apr_add_record open file:/data/log/apr/MT7-TL10_V100R001CHNC00B127SP03_Y9K0215616003102_20170716070918_statistics
    log_apr_add_record open file:/data/log/apr/MT7-TL10_V100R001CHNC00B127SP03_Y9K0215616003102_20181102231405_statistics
    move_input_files, create dir [/data/log/unzip/MT7-TL10_V100R001CHNC00B127SP03_Y9K0215616003102_20181126143244_crash]
11-26 14:32:44.111 2429-2504/? E/logserver: copy_file_to_dir, /data/log/logcache/33592936/data_app_crash@1543239164106.txt -> /data/log/unzip/MT7-TL10_V100R001CHNC00B127SP03_Y9K0215616003102_20181126143244_crash
    copy_file_to_dir, /data/log/logcache/33592936/finger.txt -> /data/log/unzip/MT7-TL10_V100R001CHNC00B127SP03_Y9K0215616003102_20181126143244_crash
    init_notify_fifo, before change /data/log/logcontrol owner
    Process 2504 opening FIFO for O_WRONLY
    Process 2504 opened fifo(19)
    notify_logcontrol, 2504 sent /data/log/unzip/MT7-TL10_V100R001CHNC00B127SP03_Y9K0215616003102_20181126143244_crash 
11-26 14:32:44.111 2429-2503/? E/logserver: thread_logcontrol, select ret = 1
11-26 14:32:44.111 2429-2504/? E/logserver: check_dir_size, dir[/data/log/coredump/] doesn't exist
11-26 14:32:44.111 2429-2503/? E/logserver: handle_notify_event, read res = 324, client pid = 2504, data=/data/log/unzip/MT7-TL10_V100R001CHNC00B127SP03_Y9K0215616003102_20181126143244_crash
11-26 14:32:44.111 2429-2504/? E/logserver: clean_cur_cache:962, system(rm -r /data/log/logcache/33592936/* > /dev/null 2>&1)
11-26 14:32:44.111 2429-2503/? E/logserver: handle_notify_event, send msg [submit:trigger=0,bugtype=2,modulename=com.brenco.citylockerapp,level=1,testtype=NORMAL,path=/data/log/unzip/MT7-TL10_V100R001CHNC00B127SP03_Y9K0215616003102_20181126143244_crash;]
    send_to_client, send to (18) res = 178
11-26 14:32:44.111 3298-3622/? E/SubmitMessage: Submit string: submit:trigger=0,bugtype=2,modulename=com.brenco.citylockerapp,level=1,testtype=NORMAL,path=/data/log/unzip/MT7-TL10_V100R001CHNC00B127SP03_Y9K0215616003102_20181126143244_crash
11-26 14:32:44.121 2429-2504/? E/logserver: Object Path:/data/log/apr/, mask=0x00000008
    event->len=80, name=MT7-TL10_V100R001CHNC00B127SP03_Y9K0215616003102_20170716070918_statistics
    process_one_event, can not find this event(MT7-TL10_V100R001CHNC00B127SP03_Y9K0215616003102_20170716070918_statistics)
    clean_cur_cache:962, system(rm -r /data/log/logcache/33592936/* > /dev/null 2>&1)
11-26 14:32:44.131 2429-2504/? E/logserver: Object Path:/data/log/apr/, mask=0x00000008
    event->len=80, name=MT7-TL10_V100R001CHNC00B127SP03_Y9K0215616003102_20181102231405_statistics
    process_one_event, can not find this event(MT7-TL10_V100R001CHNC00B127SP03_Y9K0215616003102_20181102231405_statistics)
    clean_cur_cache:962, system(rm -r /data/log/logcache/33592936/* > /dev/null 2>&1)
11-26 14:32:44.146 2919-3010/? I/PowerManagerService: canDreamLocked:mCharingBase=false
    canDreamLocked:mChargingMode=0
11-26 14:32:44.251 3265-5023/? W/anbe: Empty context buffer. Thus might mean that the context is not synced down.
11-26 14:32:44.251 3265-5023/? W/amxh: Empty context model while retrieving upload opt-in status!
11-26 14:32:44.301 2919-3010/? I/PowerManagerService: canDreamLocked:mCharingBase=false
    canDreamLocked:mChargingMode=0
11-26 14:32:44.311 3298-3586/? I/LogServiceSetting: checkMemorydumpByPath: filePath is /storage/emulated/legacy/bugreports/ramdump
    CheckMemorydumpByPath: /storage/emulated/legacy/bugreports/ramdumpis not exist.
    checkMemorydumpByPath: filePath is /storage/sdcard1/bugreports/ramdump
    CheckMemorydumpByPath: /storage/sdcard1/bugreports/ramdumpis not exist.
11-26 14:32:44.601 31335-31335/? I/HwLauncher: DynamicIcon onWindowVisibilityChanged 4 - com.android.calendar
    DynamicIcon onWindowVisibilityChanged 4 - com.android.deskclock
    DynamicIcon onWindowVisibilityChanged 4 - com.google.android.calendar
11-26 14:32:44.606 31335-31335/? I/HwLauncher: Launcher onStart()
    Launcher dynamicIconsRegister
    DynamicUpdater registerReceiver
11-26 14:32:44.611 31335-31335/? I/HwLauncher: DynamicUpdater call updateFolder
    DynamicIcon onResume  isvisible = false   mAttachedToWindow:true    mWindowVisible:falsecom.google.android.calendar
    DynamicUpdater registerReceiver
    DynamicUpdater call updateFolder
    DynamicIcon onResume  isvisible = false   mAttachedToWindow:true    mWindowVisible:falsecom.android.calendar
    DynamicUpdater registerReceiver
11-26 14:32:44.616 31335-31335/? I/HwLauncher: DynamicUpdater call updateFolder
    DynamicIcon onResume  isvisible = false   mAttachedToWindow:true    mWindowVisible:falsecom.android.deskclock
    DynamicUpdater registerReceiver
    DynamicUpdater call updateFolder
11-26 14:32:44.616 31335-31358/? I/HwLauncher: WeatherDynamicUpdaterpositiveUpdate at WeatherDynamicUpdater
11-26 14:32:44.616 31335-31335/? I/HwLauncher: DynamicIcon onResume  isvisible = true   mAttachedToWindow:false    mWindowVisible:falsecom.huawei.android.totemweather
    Launcher onResume()
    Launcher doResumeWork()
11-26 14:32:44.646 31335-31335/? W/MotionDetectionManager: startMotionAppsReco motionApps: 402 disabled
11-26 14:32:44.646 31335-31335/? W/HwLauncher: Launcher.MotionManager failed to start TYPE_SHAKE_CHANGE_WALLPAPER
11-26 14:32:44.646 31335-31335/? I/HwLauncher: Launcher.MotionManager startShakeMotion mIsShakeInHome=true
11-26 14:32:44.671 31335-31335/? W/MotionDetectionManager: startMotionAppsReco motionApps: 404 disabled
11-26 14:32:44.671 31335-31335/? W/HwLauncher: Launcher.MotionManager failed to start TYPE_SHAKE_START_PRIVACY
11-26 14:32:44.671 31335-31335/? I/HwLauncher: Launcher changeNaviBarStatus
11-26 14:32:44.671 31335-31335/? W/HwLauncher: Launcher  updateMarginLayoutParamsForView:the target view is null.
11-26 14:32:44.671 31335-31335/? I/HwLauncher: Launcher  onResume mIsToUninstallApp = false
11-26 14:32:44.676 31335-31335/? I/HwLauncher: DynamicIcon onVisibilityChanged 0 - com.android.calendar
    DynamicIcon onVisibilityChanged 0 - com.android.deskclock
    DynamicIcon onVisibilityChanged 0 - com.google.android.calendar
    DynamicIcon onWindowVisibilityChanged 0 - com.android.calendar
    DynamicIcon onWindowVisibilityChanged 0 - com.android.deskclock
    DynamicIcon onWindowVisibilityChanged 0 - com.google.android.calendar
11-26 14:32:44.731 31335-31335/? I/HwLauncher: WeatherDynamicUpdaterreceive action = com.huawei.android.action.launcher_weather_icon_change
11-26 14:32:44.731 31335-31358/? I/HwLauncher: WeatherDynamicUpdaterreceiver data is int[] = [0, 0, 4, 0, 2, 0, 0, 0]
     stringArray[] [mostlycloudy, num1, num7, tempc]
11-26 14:32:44.881 2919-3010/? I/PowerManagerService: canDreamLocked:mCharingBase=false
    canDreamLocked:mChargingMode=0
11-26 14:32:45.916 29482-29662/com.brenco.citylockerapp W/DynamiteModule: Local module descriptor class for com.google.android.gms.googlecertificates not found.
11-26 14:32:45.926 29482-29662/com.brenco.citylockerapp I/DynamiteModule: Considering local module com.google.android.gms.googlecertificates:0 and remote module com.google.android.gms.googlecertificates:4
    Selected remote version of com.google.android.gms.googlecertificates, version >= 4
1`

**Comment by waelba** [comment]
Hi, 

You can use this firebase configurations while waiting for the update of SDK.

```java
implementation 'com.google.firebase:firebase-core:11.0.4'
implementation 'com.google.firebase:firebase-auth:11.0.4'
implementation "com.google.firebase:firebase-messaging:11.0.4"
implementation 'com.google.firebase:firebase-database:11.0.4'
implementation 'com.google.firebase:firebase-storage:11.0.4'
```

**Comment by karimkod** [comment]
Hoping to get an update soon, thank you.

**Comment by waelba** [comment]
Hi @karimkod  ,
It's fixed in v1.7.1
Thanks
